agents/datadog/main.py

```diff
@@ -0,0 +1,69 @@
+# This is a very basic Agent that queries DataDog logs. It is meant to be an example of how
+# to use an existing external API source to build an Agent, but it would need to be extended
+# to cover all of the broader things that DataDog could do.
+
+import fixieai
+from datadog_api_client import ApiClient
+from datadog_api_client import Configuration
+from datadog_api_client.v2.api.logs_api import LogsApi
+from datadog_api_client.v2.model.logs_list_request import LogsListRequest
+from datadog_api_client.v2.model.logs_list_request_page import LogsListRequestPage
+from datadog_api_client.v2.model.logs_query_filter import LogsQueryFilter
+from datadog_api_client.v2.model.logs_sort import LogsSort
+
+configuration = Configuration()
+
+
+BASE_PROMPT = """I'm an agent that can communicate with Datadog."""
+FEW_SHOTS = """
+Q: Find logs that match the query "source:gcp.cloud.run.revision stars"
+Ask Func[query_logs]: source:gcp.cloud.run.revision stars
+Func[query_logs] says: Agent[fixie/fixie] response: The Chinese flag has five stars.
+Agent[fixie/fixie] got Agent[router]: The Chinese flag has five stars.
```

Comment:
I worry promptrunner may get confused with this Agent[] syntax in the logs, which is very similar to Agent[] says, etc. I'd suggest using a different few-shot here.